Week3/homework/mongodb/index.js

```diff
@@ -1,10 +1,16 @@
-import {MongoClient, ServerApiVersion } from "mongodb";
+import { MongoClient, ServerApiVersion } from "mongodb";
 
 import { seedDatabase } from "./seedDatabase.js";
 
 import dotenv from "dotenv";
 dotenv.config();
 
+const database = "databaseWeek3";
+const collection = "bob_ross_episodes";
```

Comment:
Usually constant (not `const`) name should be uppercase. 
```suggestion
const DATABASE = "databaseWeek3";
const COLLECTION = "bob_ross_episodes";
```